src/config/app.ts

```diff
@@ -295,7 +294,7 @@ function createJobsForTool(
     if (tool.executionType === ToolExecutionType.STATIC) {
         const lockedOrLatestDependencySet: ComposerDependencySet = config.lockedDependenciesExists
             ? ComposerDependencySet.LOCKED
-            : ComposerDependencySet.LATEST;
+            : ComposerDependencySet.LOWEST;
```

Comment:
same as above. If there is no lockfile, having lowest would be super weird imho.